Configuration/Templates/BLUFOR Gear Level One

```diff
@@ -0,0 +1 @@
+BluforGearLevelOne = ["rhs_weapon_m16a4", "rhs_weapon_m4_m203", "rhs_weapon_m249", "rhs_weapon_m240B", "rhs_weapon_m72a7", "rhs_weapon_M136", "rhsusf_weapon_m9", "rhsusf_weapon_glock17g4", "rhsusf_weapon_m1911a1", "rhs_weapon_m24sws", "rhs_weapon_m40a5", "Item_rhsusf_acc_wmx_bk", "Item_rhsusf_acc_ACOG", "Item_rhsusf_acc_compm4", "Item_rhsusf_acc_M8541", "Item_rhsusf_acc_harris_bipod", "rhs_magazine_rhs_grenade_anm8_mag", "Item_FirstAidKit", "Item_Medikit", "Item_ToolKit", "Item_Binocular", "rhsusf_assault_eagleiii_ocp", "rhsusf_assault_eagleiii_coy", "Item_rhsusf_lwh_helmet_marpatwd", "Item_rhsusf_ach_helmet_ocp", "Headgear_H_Hat_Tinfoil_F", "Item_rhsusf_cvc_ess", "Item_rhsusf_uniform_FROG01_wd", "Item_rhs_uniform_cu_ocp", "Item_rhsusf_iotv_ocp_Rifleman", "Item_rhsusf_iotv_ocp_Squadleader", "Item_rhsusf_spc_rifleman", "Item_rhsusf_spc_squadleader", "rhs_magazine_rhs_mag_m67", "rhs_magazine_rhs_mag_M441_HE", "rhs_magazine_rhs_mag_M714_white"]
```

Comment:
I'd prefer if we broke this out line by line, also I'd remove medkit + FAK because we are running ace. All the medical/binocs/hats etc should probably be in a default list.